R/colocalization_heatmap.R

```diff
@@ -12,6 +12,9 @@
 #' instead of drawing the heatmap
 #' @param symmetrise Set to \code{TRUE} if only the lower or upper triangle of marker combinations exist in \code{data},
 #'                   each row will then be mirrored to fill the missing triangle of the heatmap.
+#' @param legend_range A numeric vector of length 2 with the range of the legend. If NULL, the range is set to the
```

Comment:
If I understand it correctly, if values are outside of this range, the values are capped. I think it should be stated here.

Reply:
Good point! This is fixed now.